Podfile

```diff
@@ -166,7 +166,7 @@ abstract_target 'Apps' do
     ## Gutenberg (React Native)
     ## =====================
     ##
-    gutenberg :tag => 'v1.57.0-alpha1'
+    gutenberg :commit => '89f4f8caaa5291db6a55d165a4b5789e75d43eb1'
```

Comment:
> Currently, the JavaScript side of this work is only available/working if one has the Metro server running. I believe this is because we have not generated a new bundle via npm run bundle within gutenbeg-mobile, which is required for WPiOS integration. 

Good catch on that...I didn't realize I had to do anything different there. This is the first project I've worked on that required changes to both the WP apps and mobile gutenberg so I've not run into this before. 

> Additionally, you may already be aware, but once we begin the merge waterfall we will need to create a v1.57.0-alphaN tag within wordpress-mobile/gutenberg-mobile and reference that tag here.

Thank you for this additional information. I will keep that in mind. 

> All that said, I attempted to npm run bundle locally and build via Xcode to verify all the above to be true and the feature in this PR works as expected. I was unsuccessful.

I've done the same but it worked on my end. I've committed everything and requested an installable build for this PR so we'll see if it's _actually working_.